File: C#/conversor.cs
using System;

class ConversorBinario {
    static void Main() {
        Console.Write("Digite um numero decimal: ");
        int decimal_ = int.Parse(Console.ReadLine());

        if(decimal_ == 0) {
            Console.WriteLine("Binario: 0");
        }else{
            string binario = "";
            int numero = decimal_;
            while(numero>0){
                binario = (numero%2) binario;
                numero /= 2;
            }
            Console.WriteLine("Binario: " + binario);
        }
    }
}

File: Dart/conversor.dart
import 'dart:io';

String decimalParaBinario(int decimal) {
  if (decimal == 0) return '0';

  String binario = '';
  int numero = decimal;
  while (numero > 0) {
    binario = '{numero % 2}binario';
    numero ~/= 2;
  }
  return binario;
}

void main() {
  stdout.write('Digite um numero decimal: ');
  int decimal = int.parse(stdin.readLineSync()!);
  print('Binario: ${decimalParaBinario(decimal)}');
}
